hooks/environment

```diff
@@ -70,8 +70,10 @@ fi
 if [ -n "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}" ]; then
   eval "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}='${secret}'"
   export "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR?}"
+  buildkite-agent redactor add "${secret}"  # Redact the secret in Buildkite logs
 else
   variable_name=$(generate_variable_name "$BUILDKITE_PLUGIN_VAULT_SECRETS_PATH" "$BUILDKITE_PLUGIN_VAULT_SECRETS_FIELD" "$BUILDKITE_PLUGIN_VAULT_SECRETS_PATH_DEPTH")
   eval "${variable_name}='${secret}'"
   export "${variable_name?}"
+  buildkite-agent redactor add "${secret}" # Redact the secret in Buildkite logs
```

Comment:
```suggestion
  echo "${secret}" | buildkite-agent redactor add # Redact the secret in Buildkite logs
```

Reply:
Or we should make sure that there's a temporary file with the contents, then pass that in